drools-yaml-rules-benchmark/src/main/java/org/drools/yaml/benchmark/DroolsDurableBenchmark.java

```diff
@@ -1,9 +1,7 @@
 package org.drools.yaml.benchmark;
 
-import java.util.concurrent.TimeUnit;
-
-import org.drools.yaml.core.RulesExecutor;
-import org.drools.yaml.durable.DurableNotation;
+import org.drools.yaml.api.context.RulesExecutor;
```

Comment:
I'm a bit worried of the performance implication of using efesto in this project. It would be nice to have a parametric benchmark testing the engine with and without going through efesto to check its eventual impact.